src/components/Loader.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable @bigbinary/neeto/no-dangling-constants */
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect, useContext } from "react";
+import Animated, {
+  useSharedValue,
+  withTiming,
+  useAnimatedProps,
+  withRepeat,
+  Easing,
+  useAnimatedStyle,
+  interpolate,
+} from "react-native-reanimated";
+import Svg, { Circle } from "react-native-svg";
+import PropTypes from "prop-types";
+import { ThemeContext } from "styled-components/native";
+import { Container } from "@components";
+
+const LOADER_WIDTH = {
+  s: 24,
+  m: 36,
+  l: 48,
+};
+
+const AnimatedCircle = Animated.createAnimatedComponent(Circle);
+
+/**
+ ** Component is used to show a loading indication to user.
+ *
+ * <div class="screenshots">
+ *   <img src="screenshots/loader/loader.png" />
```

Comment:
@kiran-kumar011 I could not find `screenshots/loader/loader.png` in this pr.

Reply:
@sangameshsomawar ,
will remove this since in all the components in the library use this similar HTML tags because of which I thought this could be a part of the documentation.

Reply:
@kiran-kumar011 We need to take a screenshot of storybook and add `screenshots/loader/loader.png`, you can check other screenshots for reference.